utils/monitoring_tools/hardware_monitoring/README.md

```diff
@@ -0,0 +1,158 @@
+# Neptune Hardware Monitoring
+
+<div align="center">
```

Comment:
The centered tiny badge looks a bit off

Reply:
Yeah, I felt the same. Do you recommend removing it altogether, or have a better design in mind?